ubuntu-dev/ubuntu-dev.dockerfile

```diff
@@ -108,7 +108,7 @@ RUN mkdir /home/user/.ccache \
 RUN __GIT_VERSION__="2.16.2" \
  && mkdir /tmp/git \
  && cd /tmp/git \
- && curl https://www.kernel.org/pub/software/scm/git/git-${__GIT_VERSION__}.tar.xz | tar xJ \
+ && curl -L https://www.kernel.org/pub/software/scm/git/git-${__GIT_VERSION__}.tar.xz | tar xJ \
```

Comment:
Why not (also) change the URL to `https://mirrors.edge.kernel.org/pub/software/scm/git/git-${__GIT_VERSION__}.tar.xz`? This seems to be the new URL that they use, and maybe `www.kernel.org/pub/...` will stop redirecting at some point.

Reply:
https://git-scm.org still links to the current link, and the redirect may turn into local mirror some day, while the canonical link is unlikely to change.

Reply:
Ok, makes sense. Thanks! I'll approve and merge this pull request then.